pkg/discovery/worker/quotas_discovery_worker.go

```diff
@@ -77,8 +83,11 @@ func (worker *k8sResourceQuotasDiscoveryWorker) Do(quotaMetricsList []*repositor
 		// Bought resources from each node
 		// create provider entity for each node
 		for _, node := range kubeNodes {
-			nodeUID := node.UID
-			quotaEntity.AddNodeProvider(nodeUID, quotaMetrics.AllocationBoughtMap[nodeUID])
+			//Do not include the node that is not ready
+			if util.NodeIsReady(node.Node) {
```

Comment:
Can you please explain why here we don't check if the node is schedulable (comparing to the checks above)?

Reply:
Very good question Max! We still want the scheduledisabled nodes appearing in the relationship of vdc. My take is , scheduleDisabled is a normal status, it's by design. While the NotReady nodes are more like temporary down that we want to remove it from vdc, making it status unknown, etc.